src/SprykerEco/Yves/Payone/Form/CreditCardSubForm.php

```diff
@@ -19,52 +19,96 @@
 
 class CreditCardSubForm extends AbstractPayoneSubForm
 {
+    /**
+     * @var string
+     */
     public const PAYMENT_METHOD = 'credit_card';
 
+    /**
+     * @var string
+     */
     public const FIELD_CARD_TYPE = 'cardtype';
+
+    /**
+     * @var string
+     */
     public const FIELD_CARD_NUMBER = 'cardpan';
+
+    /**
+     * @var string
+     */
     public const FIELD_NAME_ON_CARD = 'cardholder';
+
+    /**
+     * @var string
+     */
     public const FIELD_CARD_EXPIRES_MONTH = 'cardexpiredate_month';
+
+    /**
+     * @var string
+     */
     public const FIELD_CARD_EXPIRES_YEAR = 'cardexpiredate_year';
+
+    /**
+     * @var string
+     */
     public const FIELD_CARD_SECURITY_CODE = 'cardcvc2';
+
+    /**
+     * @var string
+     */
     public const FIELD_PSEUDO_CARD_NUMBER = 'pseudocardpan';
 
+    /**
+     * @var string
+     */
     public const OPTION_CARD_EXPIRES_CHOICES_MONTH = 'month choices';
+
+    /**
+     * @var string
+     */
     public const OPTION_CARD_EXPIRES_CHOICES_YEAR = 'year choices';
+
+    /**
+     * @var string
+     */
     public const OPTION_CARD_TYPES = 'card types';
 
+    /**
+     * @var string
+     */
     public const OPTION_PAYONE_SETTINGS = 'payone settings';
 
     /**
      * @return string
      */
-    public function getName()
+    public function getName(): string
     {
         return PaymentTransfer::PAYONE_CREDIT_CARD;
     }
 
     /**
      * @return string
      */
-    public function getPropertyPath()
+    public function getPropertyPath(): string
     {
         return PaymentTransfer::PAYONE_CREDIT_CARD;
     }
 
     /**
      * @return string
      */
-    public function getTemplatePath()
+    public function getTemplatePath(): string
     {
-        return PayoneConstants::PROVIDER_NAME . '/' . self::PAYMENT_METHOD;
+        return PayoneConstants::PROVIDER_NAME . '/' . static::PAYMENT_METHOD;
```

Comment:
Method could be moved to AbstractPayoneSubForm

Reply:
In AbstractPayoneSubForm we don't have PAYMENT_METHOD constant. In AbstractPayoneSubForm we can move only first part on method PayoneConstants::PROVIDER_NAME and then add static::PAYMENT_METHOD into each subform. But, I'm not sure, that it makes sense.